.github/PROCESS.md

```diff
@@ -0,0 +1,15 @@
+# The Process
+
+This file contains some process notes for FerretDB Inc. engineers and long-term contributors.
+If you are a community contributor, feel free to ignore all this!
+We will do all that for you so you can focus on what is really interesting for you.
+
+1. Pull request title should be accurate and descriptive as it is used in the generated changelog.
+   It should not end with a dot.
+   It should not mention the issue number but describe the fixed issue instead.
+2. Pull request description typically just mentions the issue number like "Refs #123." or "Closes #456." (do not use `https://github.com/org/repo/issue/123` even if it is rendered the same on GitHub).
+   End sentences with a dot.
+3. Commits messages (both titles and bodies) are not important and can be "WIP" or anything else.
+   All commits are squashed on merge.
+   Autogenerated squash commit's title and body should be replaced by PR's title and description; that includes both manual merges and auto-merges.
+   Do not leave all those WIPs in the commit body.
```

Comment:
Does automatic merge removes those WIPs or not? Should I care about it when I press the button in guthub?

Reply:
No, it doesn't remove them automatically. Yes, one should care and remove them manually. :) I tried my best to explain it in the previous sentence. How it can be made better?

Reply:
suggested change.